www/templates/diary/infinite_scroll_list.html

```diff
@@ -51,48 +53,48 @@ <h3 ng-if="!infScrollControl.reachedEnd">{{'diary.scroll-to-load-more' | transla
-                            <i class="ion-more"></i>
-                        </div>
-                    </div>
-                    <div class="row">
-                        <div class="col-90" ng-click="showDetail($event, trip)">
-                            <div ng-class="listLocationClass" id="no-border" href="#" style="background-color: transparent; font-size: 0.8em; padding-top: 5px; padding-bottom: 5px; padding-left: 30px; margin-top: 0; margin-bottom: 0;">
-                            <i class="icon ion-ios-location" style="font-size: 16px; left: 0; color: #33e0bb;"></i>
-                                    {{trip.start_display_name}}
-                            </div>
-                            <div ng-class="listLocationClass" id="no-border" href="#" style="background-color: transparent; font-size: 0.8em; padding-top: 5px; padding-bottom: 5px; padding-left: 30px; margin-top: 0; margin-bottom: 0;">
-                            <i class="icon ion-ios-location" style="font-size: 16px; left: 0; color: #ff5251;"></i>
-                                    {{trip.end_display_name}}
-                            </div>
-                        </div>
-                        <verifycheck linkedtag="{{surveyOpt.elementTag}}"
-                            class="col-10 diarycheckmark-container center-vert center-horiz"></verifycheck>
-                    </div>
-                    <div class="row" style="margin-top: 0px">
-                        <!-- Assumes that the trip labels have already been
-                            filled in outside the directive to filter properly
-                            e.g. https://github.com/e-mission/e-mission-docs/issues/674#issuecomment-933083710
-                        -->
-                        <linkedsurvey element-tag="{{surveyOpt.elementTag}}" class="col" trip="trip"></linkedsurvey>
-                    </div>   
+                <infinite-scroll-trip-item trip="trip"></infinite-scroll-trip-item>
+                
+                <div ng-if="trip.display_date != trip.nextTrip.display_date && trip.nextTrip.display_date != undefined">
+                    <h2 class="hr-lines" style="justify-content: center; 
+                            font-family: 'Lato';
+                            font-weight: bold; 
+                            display: flex;
+                            color: #212121;
+                            font-size: 19px;
+                            margin-left: 10px; 
+                            margin-right: 10px;
+                            margin-bottom: 5px; 
+                            margin-top: 5px" translate> {{trip.nextTrip.display_date}} </h2>
                 </div>
-            </ion-item>
+
             </div>
-           <div class="stop-time-tag-inf-scroll">{{trip.display_end_time}}</div>
         </div>
+
+        <!-- To Handle the last diary card not being in full view -->
+        <div style="font-size: 50px; color: transparent;">SPACE</div>
+
```

Comment:
I ran it on a few different emulators, and the bottom time tag was cut off from view. To see it, I had to hold the scroll up, if that makes sense.

Reply:
@sebastianbarry is this the fix for the iOS scrolling issue? Does it actually ensure that the last diary card is visible?
Can you confirm that we don't see the word "SPACE"?

(we shouldn't, since the color is transparent, but &nbsp; might be safer all around).

Reply:
This is the fix for the iOS scrolling issue. I made the SPACE text black and can see that it does indeed cause the scrolling to stop at the proper point
Here's the SPACE in black text:
<img width="394" alt="Screen Shot 2022-09-21 at 11 54 41 AM" src="https://user-images.githubusercontent.com/61334340/191585601-95283107-ce16-4797-b01c-003ab64199cb.png">

Here is how far down the iOS scroll goes when you scroll all the way to the bottom (as you can see, the very bottom of the shadow gets a little cut-off, but I think this is perfect):
<img width="410" alt="Screen Shot 2022-09-21 at 11 55 34 AM" src="https://user-images.githubusercontent.com/61334340/191585798-bfc7669d-8303-49d2-969e-5ac4a0b77a08.png">

Reply:
@sebastianbarry 
Does this still look good on android? If this solution works fine on both android and iOS, could you file an issue to make a similar change for the scrolling on the permission screen?

that's the other place that people (including you) have complained about.

> When asking for permission in the beginning, I could not scroll (Using iPhone 8)
> Can't label rides at the bottom of the page without using two hands (One finger to scroll and on to tap while holding the screen). It basically automatically goes back up.

Reply:
If this messes things up on android, you could add an `ng-if` so that this extra space only appears on iOS.

Reply:
This works on my Android Pixel 6!

<img width="1019" alt="Screen Shot 2022-09-23 at 12 43 41 PM" src="https://user-images.githubusercontent.com/61334340/192035911-f4d260c0-aff4-4669-a490-13497bfb2f3c.png">

This looks good to go to me. The android does not scroll farther than this, it doesn't have the option to drag and hold like the iPhone does either.

_**Confirmed this change as complete**_